templates/space/_filters.html.twig

```diff
@@ -1,24 +1,23 @@
-<button class="returnButton" id="myModalBtn" type="submit">{% include "space/_filter_button.html.twig" %}</button>
+<span class="returnButton" id="myModalBtn">{% include "space/_filter_button.html.twig" %}</span>
 
 <div
 	id="myModal" class="modal">
 	<!-- Modal content -->
 	<div class="modal-content">
-		<form class="form-filters">
```

Comment:
the opening tag should be deleted. if you need to have inputs in the same form, put them in the same form, i.e. in the same file

Reply:
done

Reply:
how can this be done since you're still deleting the tags? restore what you destroyed. by the way, you created a new form without an ending tag

Reply:
i removed the filters which is now useless and closed form

Reply:
fix indentation and check again opening/closing tags in every file, and don't push until you've done your work for real

Reply:
seems good

Reply:
clearly not. @Superzut44 @N26N26N26 @Marionptnt one of you takes the task